tools/src/launcher/Main.hs

```diff
@@ -510,10 +506,10 @@ spawnNode (path, args, mbLogPath) = do
             -- fails when it prints unicode characters. In the meantime, don't print it.
             -- See DAEF-12.
             -- printf ("Redirecting node's stdout and stderr to "%fp%"\n") logPath
-            return $ createProc PHandle path args (Just logHandle)
-        Nothing -> return $ createProc PInherit path args Nothing
+            return $ createProc (Process.UseHandle logHandle) path args
```

Comment:
Do we even need this anymore ? Maybe I can just do `createProc Process.Inherit path args` and delete this as well as node and wallet log path fields if the nodes already print log to file by default.

Reply:
It's better to leave this code. Node and wallet can log to file, but:
1. It's not default behaviour, it depends on logging config. We pass config which makes them print to file, but we may forget or something.
2. Not all output is done via log-warper. For instance, if an uncaught exception happens, it will be printed to `stderr` by runtime system. So the only way to make it appear in a file is to redirect `stderr` to this file.